src/pages/ShowResource.tsx

```diff
@@ -18,46 +20,71 @@ function ShowResource({ location }: RouteComponentProps) {
 
   const { data: entity, isLoading, error, reload } = useAsync(
     getComitResourceFn,
-    { resourcePath, watch: location.pathname }
+    {
+      resourcePath,
+      watch: location.pathname
+    }
   );
+  const axiosError = error as AxiosError;
 
-  if (isLoading) {
-    return <CenteredProgress title="Fetching..." />;
-  } else if (
-    !error &&
-    entity &&
-    entity.class &&
-    entity.class.includes("swaps")
-  ) {
-    return (
+  const [showLoading, setShowLoading] = useState(false);
```

Comment:
I still believe we don't need this. Can you try to remove it?

Reply:
I tried to remove it and remembered why I added it: the `autoHideDuration` prop of Material UI's snackbar only does something if `onClose` is defined.